tests/test_parse.py

```diff
@@ -28,14 +28,19 @@ def test_model_validate_submodel():
     assert m.model_dump() == {'a': 10.2, 'b': 10}
 
 
-@pytest.mark.xfail(reason='working on V2')
 def test_model_validate_wrong_model():
```

Comment:
This one's not really related, but it was another one that just seemed to need an update to the output to be in v2 convention.